### DISCUSSION TITLE
Opt-out requirement to reset admin password after init

### DISCUSSION THREAD

**Discussion by ghost** [zitadel/zitadel#3951] [Ideas]
As my setup already stores the default admin password securely, there is no need to force a password reset. It's not a breaking change as Zitadel's state already has to be stored, but it is unnecessary.

**Comment by fforootd** [comment]
Hi @thegergo02 

I think you can skip that by changing[ this value ](https://github.com/zitadel/zitadel/blob/v2-alpha/cmd/setup/steps.yaml#L22)

`PasswordChangeRequired: true`
